package.json

```
@@ -27,5 +27,9 @@
   "engines": {
     "node": "^18.12.1"
   },
-  "packageManager": "yarn@3.3.0"
+  "packageManager": "yarn@3.3.0",
+  "dependencies": {
+    "@prisma/client": "latest",
+    "prisma": "latest"
+  }
```

your installing packages in the project root, i recommend you read this https://yarnpkg.com/cli/workspace
```suggestion
  "packageManager": "yarn@3.3.0"
```
(The version is also latest, which i'm strongly against but besides the point here.)

---

(fixed)